packages/kbn-generate-console-definitions/src/types/specification_types.ts

```diff
@@ -0,0 +1,442 @@
+/*
+ * Copyright Elasticsearch B.V. and/or licensed to Elasticsearch B.V. under one
+ * or more contributor license agreements. Licensed under the Elastic License
+ * 2.0 and the Server Side Public License, v 1; you may not use this file except
+ * in compliance with, at your election, the Elastic License 2.0 or the Server
+ * Side Public License, v 1.
+ */
+
+/**
+ * --------------- THIS FILE IS COPIED FROM ES SPECIFICATION REPO -------------------
```

Comment:
I added a task to the meta issue to see if we could import the types instead of copying